README.md

```diff
@@ -476,5 +484,52 @@ art.artifacts.delete("<ARTIFACT_PATH_IN_ARTIFACTORY>")
 ```
 
 
+
+### Builds
+
+#### Get a list of all builds
+```python
+build_list: BuildListResponse = art.builds.list()
+```
+
+#### Get the information about a build
+
+```python
+build_info: BuildInfo = art.builds.get_build_info("<build_name>", "<build_number>")
+```
+
+Note: optional BuildProperties can be used to query the correct build info of interest.
+
+#### Create build
+
+```python
+_build_create_request = BuildCreateRequest(name="<build_name>", number="<build_number>", started="<Build start time in the format of yyyy-MM-dd'T'HH:mm:ss.SSSZ>")
+_create_build = art.builds.create_build(_build_create_request)
+```
+
+
+#### Promote a build
+```python
+_build_promote_request = BuildPromotionRequest(sourceRepo="<source-jfrog-repo>", targetRepo="<target-jfrog-repo>")
```

Comment:
Why the underscore? I don't think it's necessary

Reply:
I'm used to, I'll remove them